### PR TITLE
Fix typo and update doc

### PR DESCRIPTION
**Pull request by Gallardot**
### What problem does this PR solve? 

Fix typo for Unit test
Update doc for helm

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: helm/chaos-mesh/values.yaml [hunk 1]
```diff
@@ -70,6 +70,18 @@ chaosDaemon:
   # runtime: containerd
   # socketPath: /run/containerd/containerd.sock
 
+  resources:
```
Comment (YangKeao) [inline helm/chaos-mesh/values.yaml]: Should we remove these resource limits rather than add them for all? @cwen0

Comment (Gallardot) [reply helm/chaos-mesh/values.yaml]: I agree with you. I think `resources:{}` is OK.

File: doc/deploy.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ kubectl get pods --namespace chaos-testing -l app.kubernetes.io/instance=chaos-m
 
 > **Note:**
 >
-> Due to current development status of Chaos Dashboard, it is not installed by default. If you want to try it out, add `--set dashboard.create=true` in the helm commands above.
+> Due to current development status of Chaos Dashboard, it is not installed by default. If you want to try it out, add `--set dashboard.create=true` in the helm commands above.Please refer to [Configuration](../helm/chaos-mesh/README.md#parameters) for more information.
```
Comment (fewdan) [inline doc/deploy.md]: Would you please add a space between "above.Please"?

Comment (Gallardot) [reply doc/deploy.md]: Thank you. Done.

**Pull request merged**